src/components/appComponents/booksComponents/AddBookForm.js

```diff
@@ -1,20 +1,36 @@
-import React from 'react';
+import React, { useState } from 'react';
+import { useDispatch } from 'react-redux';
+import { addBook } from '../../../redux/books/books';
 
 function AddBookForm() {
+  const dispatch = useDispatch();
+  const [formState, setFormState] = useState({ title: '', author: '', category: 'Category' });
+
+  function handleChange(e) {
+    setFormState({ ...formState, [e.target.name]: e.target.value });
+  }
+
+  function handleSubmit(e) {
+    dispatch(addBook(formState));
```

Comment:
- [ ] It will be nice to have your handleSubmit generate a unique id for a book and add it to the inputs from the state before dispatching the complete book object.